### DISCUSSION TITLE
Need a little help with a error msg.

### DISCUSSION THREAD

**Discussion by Marduc666** [Ladder99/fanuc-driver#79] [Q&A]
Hello.

I followed this guide here: https://youtu.be/gqfBj0ImMrw
But im getting an error that i cant seem to fix on my own with the help of google.
Also it was mentioned in the video that a template would maybe be posted for influx, that would be awesome if there is one i can use.

Im still trying to orient myself around github, feel kinda lost atm :)

![error_dozzle](https://user-images.githubusercontent.com/109043445/207104982-eb74098c-a79a-44cb-89e9-02f4c45f64e9.png)

Thx in Advance!

**Comment by MRIIOT** [comment]
Sorry for the late reply @Marduc666 .  Can you share your fanuc-driver config files please?
